docs/source/api/stats.rst

```diff
@@ -5,4 +5,21 @@ Statistics and diagnostics are delegated to the
 `ArviZ <https://arviz-devs.github.io/arviz/index.html>`_.
 library, a general purpose library for
 "exploratory analysis of Bayesian models."
-Refer to its documentation to use the diagnostics functions directly.
+For statistics and diagnostics, ``pymc3.<function>`` are now aliases
+for ArviZ functions. Thus, the links below will redirect you to
+ArviZ docs:
+
+.. currentmodule:: pymc3.stats
+
+
+- :func:`pymc3.bfmi <arviz:arviz.bfmi>`
+- :func:`pymc3.compare <arviz:arviz.compare>`
+- :func:`pymc3.ess <arviz:arviz.ess>`
+- :data:`pymc3.geweke <arviz:arviz.geweke>`
```

Comment:
we deleted geweke in ArviZ, it had basically been broken for a while (worked only on 1d arrays) and nobody had complained. This is why there are questions on discourse about it being missing, but so far nobody has complained about it being missing, only about the import error it causes with old pymc3 versions.